cmd/vic-machine/create/create.go

```diff
@@ -743,10 +743,16 @@ func (c *Create) processNetwork(network *data.NetworkConfig, netName, pgName, st
 
 // processDNSServers parses DNS servers used for client, external, mgmt networks
 func (c *Create) processDNSServers() error {
-	if len(c.dns) == 0 {
+	if c.Data.ClientNetwork.Empty() && c.Data.ExternalNetwork.Empty() && c.Data.ManagementNetwork.Empty() {
+		log.Warn("Specified DNS servers are ignored if static IP is not set on any networks. VCH will use DNS servers provided by DHCP.")
```

Comment:
That is not the case. I fixed this about a week ago. Now you can specify dns server while not specifying a static ip.